internal/backends/qt/QtThirdPartySoftware_Listing.txt.license

```diff
@@ -0,0 +1,2 @@
+SPDX-FileCopyrightText: 2023 The Qt Company <https://qt.io/>
```

Comment:
btw, that file is probably outdated.  We use a newer version of Qt by now, it might have new licenses to it :-/
I whish there was a way for our CI to get this file instead.